tests/wpt/meta/page-visibility/unload.html.ini

```diff
@@ -0,0 +1,4 @@
+[unload.html]
+  expected: TIMEOUT
+  [visibilitychange fires on unload]
+    expected: TIMEOUT
```

Comment:
I think these two unload ones should be able to pass, since the unload code is straight forward.

Reply:
I'll investigate what's happening.

Reply:
It looks like we do need `#make-active`. But perhaps we should discuss how to update all these in next week's meeting.
This PR became a feature creep IMHO. We should discuss how to deal with outdated steps of history session before adding `update_visibility_state` into them.

Reply:
I think we need an issue similar to https://github.com/servo/servo/issues/31182 with a little investigation for each tests, and also perhaps some architectural changes needed. Yes let's discuss next week.

Reply:
I opened #32742 and I'll look into #32687 next to see if we should fix the loading steps first.